AlphaWallet/EtherClient/Requests/EstimateGasRequest.swift

```diff
@@ -21,7 +21,7 @@ struct EstimateGasRequest: JSONRPCKit.Request {
             [
                 "from": from.description,
                 "to": to?.description ?? "",
-                "value": value.description.hexEncoded,
+                "value": "0x" + String(value, radix: 16) ?? "0x0",
```

Comment:
I think `String(_,radix:)` here will never be nil, so we can just do:

```
String(value, radix: 16).hexEncoded
```

Reply:
Cannot, if the string value is "0" it will convert it to 0x30 because 0x30 means 0 on the ascii chart...